src/sidebar/components/Annotation/AnnotationPublishControl.js

```diff
@@ -73,7 +73,7 @@ function AnnotationPublishControl({
 
   const menuLabel = (
     <div className="annotation-publish-button__menu-label" style={buttonStyle}>
-      <Icon name="expand-menu" classes="u-icon--small" />
+      <Icon name="expand-menu" />
```

Comment:
Turns out the pre-existing icon-sizing class was being superseded by a `Menu`-related class anyway.